bin/console.picocom

```diff
@@ -22,4 +22,21 @@ if [ -z "${console_baud}" ]; then
     console_baud=115200
 fi
 
+# Some boards have a UART built into them and it doesn't appear until the
+# board is powered on. An example is zynq_zybo. Try to handle this by waiting
+# for up to about 10 seconds for the console and then continuing as soon as
+# possible (so we don't miss the U-Boot banner).
```

Comment:
Note that the console script is executed before anything else, so there should be a guaranteed target reset (and indeed flash operation) before U-Boot boots, so this concern shouldn't be an issue.